OOPintro/RationalNimber.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public struct RationalNumber
+{
+    public int Numerator { get; private set; }
+    public uint Denominator { get; private set; }
+    public double Rational => ((double)Numerator) / ((double)Denominator);
```

Comment:
ЛИшние круглые скобки. К `double` достаточно привести только один операнд (любой).
```suggestion
    public double Rational => (double)Numerator / Denominator;
```